### PR TITLE
Feature/api endpoint

### PR DESCRIPTION
**Pull request by Summer0410**
Implemented basic api endpoint, which enables front end to query data from the mongodb database.

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -6,19 +6,22 @@
...
+                            sensor_status=models.SensorStatus.ONLINE,
+                            organization=org,
+                            machine=machine)
     sensor.save()
     return sensor.to_json()
 
-
 if __name__ == '__main__':
-    app.run(port=8822)
+    app.run(port=8822, debug = True)
```
Comment (BenxinNiu) [inline src/app.py]: pls remove `debug` flag

**Review by BenxinNiu — CHANGES_REQUESTED** [review]

File: src/controllers/sensor_info_controller.py [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
+    elif(status == "OFFLINE"):
+        sensors = models.Sensor.objects.filter(organization__organization_name = org_name).filter(sensor_status = models.SensorStatus.OFFLINE)
+    elif(status == "DECOMMISSIONED"):
+        sensors = models.Sensor.objects.filter(organization__organization_name = org_name).filter(sensor_status = models.SensorStatus.DECOMMISSIONED)
+    return json.loads(sensors.to_json())
+  
+def get_all_sensors_machine(org_name, machine_name):
+    logging.info("HTTP request received with org_name: %s machine_name: %s".format(org_name, machine_name))
+    sensors = models.Sensor.objects.filter(organization__organization_name = org_name).filter(machine__machine_name = machine_name)
+    return json.loads(sensors.to_json())
```
Comment (BenxinNiu) [inline src/controllers/sensor_info_controller.py:20]: note for future improvement, nothing major: extract query logic to methods in a different package to improve code readability

**Comment by BenxinNiu** [comment]
also, conflict with dev branch

**Comment by Summer0410** [comment]
Made changes to the pr, @NoahGallant-MUN  I tried to append object count at the end, did not get it working, but the return result is pretty much what you wanted other than the sensor count.  If anyone knows how to add the count at the end, please go ahead.

**Review by BenxinNiu — APPROVED** [review]

**Comment by BenxinNiu** [comment]
merged with a flag for implementing mapper between Mongo Document and Swagger object (I'll implement this).

**Pull request merged**